lib/help/defend.txt

```diff
@@ -299,69 +414,85 @@ of Speed
      
 Dragon Scale Mails.
      These extremely rare pieces of armor come in many different
-     colors, each protecting you against the relevant dragons.
-     Naturally they are all resistant to acid damage. They also
-     occasionally allow you to breathe as a dragon would!
-
-Dragon Shields and Helms
-     These shields and helms have a chance of granting one or more 
-     random resistances.
-
-Other items
-     Apart from these there are some very rare, and well made armor
-     in the dungeon with not necessarily any special abilities. These
-     include Adamantite Plate Mail, Mithril Plate Mail, Mithril Chain
-     Mail, Shields of Deflection, and Shadow Cloaks. The first four
-     cannot be damaged by acid because of the quality metals they
-     contain. Shadow Cloaks grant resistance to both light and darkness
-     attacks.
+     colors, each protecting you against the dragons of that color.
+     They can not be damaged by acid, fire, cold or electricity.
+     They also occasionally allow you to breathe as a dragon would!
+
+Dragon Shields, Helms, Boots and Gloves
+     These items have a chance of granting one or more random
+     resistances. They all can not be damaged by acid, fire, cold
+     or electricity.
+
+--- Other items ---
+
+
+Apart from these there are some very rare, and well made, pieces of
+armor in the dungeon. Adamantite Plate Mail, Mithril Plate Mail,
+and Mithril Chain Mail are not damaged by acid because of the quality
+metals they contain. Mirror Shields have the ability to reflect bolt
+spells. Elven Cloaks make the wearer more stealthy and make it easier
+to find traps or secret doors. Shadow Cloaks provide resistance to
+light and darkness attacks. All of those could also be ego items or
+artifacts.
+
+There are also legendary pieces of armor (artifacts) which are similar
+to ego items but usually have more abilities and are very powerful.
+Artifacts never break. However, they may deteriorate and, if caught
+in the effect range of *destruction, may disappear forever.
+
+In Hengband, there are two types of artifacts: fixed and random.
+A fixed artifact has a predetermined name and attributes, so,
+if you play the game many times, you may encounter the same fixed
+artifact. The name and attributes of a random artifact are determined
+when it is generated, so it is extremely unlikely that you would see
+that artifact again in another game. A fixed artifact will have a
+"★" symol in its name. A random one will have a "☆" in its name.
```

Comment:
English version doesn't add ★ and ☆ symbols to artifact header.

Reply:
I removed the sentences mentioning ★ and ☆ for artifacts.  I'll have to fix up the similar line that was included in attack.txt in a previous pull request.

I added a line to footer.  Should the Japanese help files that have had substantive recent changes be changed as well? The last line in jdefend.txt says it was updated for 1.0.11; jattack.txt says it was updated for 1.5.2.